runtime/redirector/CMakeLists.txt

```diff
@@ -43,7 +42,7 @@ target_include_directories(jvm_redirect
 		"${j9vm_SOURCE_DIR}/j9vm"
 		"${j9vm_BINARY_DIR}/j9vm"
 )
-target_compile_definitions(jvm_redirect PRIVATE J9VM_JAVA_VERSION=148)
+target_compile_definitions(jvm_redirect PRIVATE J9VM_JAVA9_BUILD=148)
```

Comment:
Are the changes in this file to change from J9VM_JAVA_VERSION to J9VM_JAVA9_BUILD a bug fix, not directly related to the other changes?

Reply:
This is an unrelated bug fix. The redirector code cares about `J9VM_JAVA9_BUILD`.